presto-orc/src/main/java/com/facebook/presto/orc/StripeReader.java

```diff
@@ -322,20 +324,29 @@ private boolean addIncludedStreams(Map<Integer, ColumnEncoding> columnEncodings,
         return hasRowGroupDictionary;
     }
 
-    private Map<StreamId, OrcInputStream> readDiskRanges(
+    private Map<StreamId, OrcInputStream> readStreamData(
```

Comment:
Any reason we wanna change the method name? `readDiskRanges` actually is a good correspondent for `getDiskRanges`